en/index.md

```diff
@@ -53,6 +53,61 @@ technology terms:
  - space-separated, comma-delimited
  - load balancer (only upper case as part of proper name e.g. Elastic Load Balancer)
 
+## Branding
+
+Consistency in branding is important for a number of reasons, including the protection of trademarks where they apply. 
+
+### Ubuntu
+
+When we refer to Ubuntu, we are usually referring to the distribution, rather than the Ubuntu project itself.
+
+Our convention is to use the name, followed by the release number and, if applicable, 'LTS' to denote that the version is in Long Term Support. Optionally, you may also further identify the release by it's codename (the first or both parts of the release name) if this is likely to be useful (e.g. some of our products also use the release names to identify versions).
+
+Some examples of correct usage:
+
+**Ubuntu 22.04 LTS**
+**Ubuntu 22.10**
+**Ubuntu 23.04 (Lunar Lobster)** (note case)
+**Ubuntu 23.10 (Mantic)**
+
+This also follows on to more specific products, e.g.
+
+**Ubuntu Server 22.04 LTS**
+
+In cases where the release may be mentioned many times in a single document, it is up to the author's judgement whether this could be reworded or replaced by simply 'Ubuntu' on subsequent mentions. 
+
+### Other Canonical products
```

Comment:
I don't mind it living ”elsewhere", as long as that is an additional page in this repo. I think it is important to have a local source for this, as it will be tied to the rules also

Reply:
This table doesn't seem opressively long to me (we have longer tables) - so I have a new suggestion - we keep 'common' or 'popular' ones which are commonly misused here and link to a full list if/when more are added. 

Thoughts?

Reply:
This is more of a product management responsibility - not the documentation team's, but I agree that it is convenient to have a list here.

Reply:
Okay, well, if and when someone manages a list in the future we can add a link.